Orange/data/dask.py

```diff
@@ -160,9 +193,27 @@ def __getitem__(self, key):
 
     def save(self, filename):
         # the following code works with both Table and DaskTable
+        def parse(attr):
+            params = (attr.name, attr.TYPE_HEADERS[1], {"attributes": attr.attributes})
+            if isinstance(attr, DiscreteVariable):
+                params[2].update(values=attr.values)
+            elif isinstance(attr, TimeVariable):
+                params[2].update(have_date=attr.have_date,
+                                 have_time=attr.have_time)
+            elif isinstance(attr, ContinuousVariable):
+                params[2].update(number_of_decimals=attr.number_of_decimals)
```

Comment:
Doesn't ContinuousVariable have more attributes on how to display stuff than the number of decimals? Maybe I mis-remember.

Reply:
I believe there is one more, but it's calculated from `number_of_decimals`.